tests/tests/test-oauth2.spec.js

```diff
@@ -0,0 +1,155 @@
+/// <reference types="cypress" />
+import 'cypress-file-upload';
+
+const CLIENT_NAME = 'test-oauth2-client';
+
+context('Create a OAuth2 Client', () => {
+  before(() => {
+    cy.loginAndSelectCluster();
+    cy.goToNamespaceDetails();
+  });
+
+  it('Navigate to OAuth2 Client', () => {
+    cy.getLeftNav()
+      .contains('Configuration')
+      .click();
+
+    cy.getLeftNav()
+      .contains('OAuth2 Clients')
+      .click();
+  });
+
+  it('Create a Client', () => {
+    cy.getIframeBody()
+      .contains('Create OAuth2 Client')
+      .click();
+
+    cy.getIframeBody()
+      .contains('Advanced')
+      .click();
+
+    cy.getIframeBody()
+      .find('.advanced-form')
+      .find('[placeholder="OAuth2 Client Name"]')
+      .clear()
+      .type(CLIENT_NAME);
+
+    cy.getIframeBody()
+      .find('.advanced-form')
+      .contains('label', 'ID Token')
```

Comment:
We could use 
```suggestion
      .contains('label', 'ID Token')
      .filter(':visible', { log: false })
```
instead to not use the class selector.